c2corg_api/models/feed.py

```diff
@@ -367,6 +401,53 @@ def update_feed_association_update(
     update_participants_of_outing(child_document_id, user_id)
 
 
+def update_feed_images_upload(images, images_in, user_id):
```

Comment:
`images` are image entities, `images_in` is what was received in the request data. This naming schema `document/document_in` is also used elsewhere.